strategy-offline-fallback/service-worker.js

```diff
@@ -0,0 +1,71 @@
+var CACHE = 'offline-fallback';
+
+// On install, cache the non available resource.
+self.addEventListener('install', function(evt) {
+  console.log('The service worker is being installed.');
+  evt.waitUntil(precache().then(function () {
+    // Skip waiting is necessary since we want the SW to be active ASAP
+    // instead of waiting for non controlled clients to be closed first.
+    return self.skipWaiting();
+  }));
+});
+
+self.addEventListener('activate', function (evt) {
+  // `self.clients.claim()` allows the service worker to start intercepting
+  // requests immediately. In addition to `self.skipWaiting()` is needed to
+  // allow serving fallbacks since the beginning.
+  evt.waitUntil(self.clients.claim());
+});
+
+// Use any strategy you want but check for errors to finally use the fallback.
```

Comment:
comma after `want`

I don't understand the rest of the comment. Do you mean the developer must check for errors before using the fallback? Or that they must check for errors and use the fallback in that case?